sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/WriteStatsTracker.scala

```diff
@@ -103,5 +103,7 @@ trait WriteJobStatsTracker extends Serializable {
    * to the expected derived type when implementing this method in a derived class.
    * The framework will make sure to call this with the right arguments.
    */
-  def processStats(stats: Seq[WriteTaskStats]): Unit
+  def processStats(stats: Seq[WriteTaskStats], jobCommitDuration: Long): Unit
+
+  def updateTaskWriteAndCommitDuration(duration: Long): Unit
```

Comment:
It's unnecessary to break custom `WriteJobStatsTracker` implementations. Can we propagate the task commit time metrics with a different way? like via `WriteJobDescription`

Reply:
same to the job commit time metrics.